panel/src/components/Dialogs/Dialog.vue

```diff
@@ -147,12 +148,27 @@ export default {
     }
   },
   methods: {
+    /**
+     * Reacts to the overlay being closed
+     * and cleans up the dialog events
+     * @private
+     */
+    onOverlayClose() {
+      this.notification = null;
+      /**
+       * This event is triggered when the dialog is being closed.
+       * This happens independently from the cancel event.
+       * @event close
+       */
+      this.$emit("close");
+      this.$events.$off("keydown.esc", this.close);
+      this.$store.dispatch("dialog", false);
+    },
     /**
      * Opens the dialog and triggers the `@open` event
      * @public
      */
     open() {
-      this.$store.dispatch("dialog", true);
```

Comment:
We are not setting it to open anymore but above still setting the store when closing?

Reply:
Let me give that a closer look again.